### PR TITLE
Ingrid kwok patch 10

### PR DESCRIPTION
**Pull request by IngridKwok**
## Summary

Please merge to main and publish release note. 

**Release**:
- [x ] When ready

--------------------------------------------------

### CODE REVIEW

**Comment by IngridKwok** [comment]
@stevector @rachelwhitton Ready to publish after your final review.

File: source/releasenotes/2024-06-04-release-note-multi-select-filter.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Multi-select filter now available for Pantheon release notes
+published_date: "2024-06-04"
+categories: [documentation, user-interface]
+---
+
+We're introducing another enhancement to our release notes: the multi-select filter. Now, you can filter release notes using multiple tags simultaneously, providing a more customized and efficient way to find relevant updates. Additionally, you can clear all filters at once with a single click.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-06-04-release-note-multi-select-filter.md:7]: I believe an extra line below this is needed to add another paragraph.

I wouldn't call out the clear filter button - that's a product spec but not a noteworthy part of this release.

**Review by scottbuscemi — DISMISSED** [review]

File: source/releasenotes/2024-06-04-release-note-multi-select-filter.md (hunk 1 above)
Comment (stevector) [inline source/releasenotes/2024-06-04-release-note-multi-select-filter.md:7]:
```suggestion
We're introducing another enhancement to our release notes: the multi-select filter. Now, you can filter release notes using multiple tags simultaneously, providing a more customized and efficient way to find relevant updates.
```

I agree we don't need to call that out

File: source/releasenotes/2024-06-04-release-note-multi-select-filter.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Explore this new functionality and streamline your experience with Pantheon release notes.
```
Comment (stevector) [inline source/releasenotes/2024-06-04-release-note-multi-select-filter.md:8]:
```suggestion

Explore this new functionality and streamline your experience with Pantheon release notes.
```

File: source/releasenotes/release-note-multi-select-filter.png
Comment (stevector) [inline source/releasenotes/release-note-multi-select-filter.png:1]: It appears that this image is not used, only the gif, right @IngridKwok? If so, let's delete it.

File: source/images/release-note-multi-select-filter.png
Comment (stevector) [inline source/images/release-note-multi-select-filter.png:1]: It appears that this image is not used, only the gif, right @IngridKwok? If so, let's delete it.

**Review by stevector — CHANGES_REQUESTED** [review]

File: source/releasenotes/release-note-multi-select-filter.png
Comment (IngridKwok) [reply source/releasenotes/release-note-multi-select-filter.png:1]: Only gif is used. @stevector

**Comment by IngridKwok** [comment]
@stevector Ready to publish.

**Comment by stevector** [comment]
@IngridKwok there are still open comments and suggestions. Can you respond to those?

File: source/releasenotes/release-note-multi-select-filter.png
[the remark this answers is not in the document]
Comment (IngridKwok) [reply source/releasenotes/release-note-multi-select-filter.png:1]: @stevector I don't see the option to delete this though. Are you able to delete it?

Comment (stevector) [inline source/releasenotes/release-note-multi-select-filter.png:1]: @IngridKwok is this `.png` used anywhere?

Comment (IngridKwok) [reply source/releasenotes/release-note-multi-select-filter.png:1]: No, the multi select filter png was supposed to be used for this release note only, but then I replaced it with the gif so only the gif should be kept. I still couldn't see the option for me to delete the png though.

**Review by stevector — APPROVED** [review]

**Pull request merged**